<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderForm</class>
 <widget class="QWidget" name="OrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>296</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>584</width>
    <height>296</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>584</width>
    <height>296</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Order</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QPushButton" name="OrderBut">
      <property name="text">
       <string>Отобразить заказы</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
